.changeset/twenty-panthers-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components-development-kitchen': minor
```

Comment:
I think this should be `major` as it can have a visual impact for consumers of the package.